apps/meteor/app/livechat/lib/messageTypes.ts

```diff
@@ -122,7 +126,10 @@ MessageTypes.registerType({
 	id: 'omnichannel_on_hold_chat_resumed',
 	system: true,
 	message: 'Omnichannel_on_hold_chat_resumed',
-	data(message) {
+	data(message: IOmnichannelSystemMessage) {
+		if (!message.comment) {
+			return;
```

Comment:
```suggestion
			return {  };
```
The translation string expects an object with a comment prop from here, so I'd not recommend returning undefined values. You may either return a empty blank string (like `{ comment: '' }`) or you can even add a custom message there (like `{ comment: 'No comment provided' }`)

Reply:
Fixed.